ros2_ws/src/otto_deliberation/src/deliberation_otto_node.cpp

```diff
@@ -61,65 +57,96 @@ class OttoDeliberation
         m_deliberationStatePublisher = m_node->create_publisher<std_msgs::msg::String>("deliberation_state", 10);
     }
 
-    template<typename Container>
-    Container filterEmptyElements(const Container& container)
+    bool Initialize()
     {
-        Container filteredContainer;
-        using T = typename Container::value_type;
-        std::copy_if(
-            container.begin(),
-            container.end(),
-            std::back_inserter(filteredContainer),
-            [](const T element)
+        const std::string assigned_lane_name = m_node->declare_parameter<std::string>("assigned_lane", "");
```

Comment:
Nit: use _camelCase_

Reply:
To be honest, I had a problem with this. I observe common approach (in our codebase), where snake case is used for rclcpp related / using code. Probably there is no good way out of this. However, I can obviously unify everything to camelCase.